src/open-hackathon-server/Kaiyuanshe.OpenHackathon.Server/Controllers/UserController.cs

```diff
@@ -65,5 +67,24 @@ public class UserController : HackathonControllerBase
             return Ok(userInfo);
         }
         #endregion
+
+        #region GetUploadToken
+        /// <summary>
+        /// Get file upload token
+        /// </summary>
+        /// <returns>The team</returns>
+        /// <response code="200">Success. The response return a token.</response>
+        [HttpPut]

```

Comment:
I prefer to use `HttpPut` for creating something new in backend and it's querable. Better to use `HttpPost` to represent an action